exporter/sync_committees_count.go

```diff
@@ -26,8 +27,7 @@ func exportSyncCommitteesCount() error {
 		return err
 	}
 
-	currEpoch := utils.TimeToEpoch(time.Now())
-	currentPeriod := utils.SyncPeriodOfEpoch(uint64(currEpoch))
+	currentPeriod := utils.SyncPeriodOfEpoch(services.LatestFinalizedEpoch())
```

Comment:
Not really a fan of the exporter depending on data from the frontend service. Maybe latestFinalizedEpoch data should be exported by the exporter instead?

Reply:
change to `db.GetLatestFinalizedEpoch()`